src/chrome/content/rules/Etiqa.com.my.xml

```diff
@@ -41,6 +41,25 @@
 
 	<securecookie host=".+" name=".+" />
 
+	<!-- only MyAccount & getonline directories are functional -->
+	<exclusion pattern="^http://(www\.)?etiqa\.com\.my/(?!getonline|MyAccount)" />
+
+		<test url="http://etiqa.com.my/" />
+		<test url="http://www.etiqa.com.my/" />
```

Comment:
They are already tested (all targets are implicit test urls). Duplicated test urls should be avoided.